pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -362,6 +362,54 @@ func fixLibvirtCrcNetworkAvailable() error {
 	return nil
 }
 
+func removeLibvirtCrcNetwork() error {
+	logging.Debug("Removing libvirt 'crc' network")
+	_, _, err := crcos.RunWithDefaultLocale("virsh", "--connect", "qemu:///system", "net-info", libvirt.DefaultNetwork)
+	if err != nil {
+		// Ignore since no crc network exists for libvirt
+		return nil
+	}
+	_, stderr, err := crcos.RunWithDefaultLocale("virsh", "--connect", "qemu:///system", "net-destroy", libvirt.DefaultNetwork)
+	if err != nil {
+		logging.Debugf("%v : %s", err, stderr)
```

Comment:
We accept setups where the `crc` network does not exist (to be honest, I'm not sure how such a setup would work), but we don't accept setups where a `crc` network exists but was stopped manually? It's hard to know where to draw the line between "this could be a valid user setup" and "we are not going to support this" :(

Reply:
@cfergeau this check is only present if a user manually delete the `~./crc` or `~/.crc/machine` directory and don't want to use crc anymore so he can clean up those using `cleanup` command.

Reply:
Removing `~/.crc` is not going to change anything with respect to libvirt network definitions.

Reply:
@cfergeau sorry, I didn't get what you are suggesting in term of destroying the crc network, do you mean we shouldn't destroy it and just undefine it? My understanding if you directly `undefine` the network without `destroy` then it will be still `active` but transient. 

```
$ sudo virsh net-list
 Name              State    Autostart   Persistent
----------------------------------------------------
 crc               active   yes         yes

$ sudo virsh net-undefine crc
Network crc has been undefined

$ sudo virsh net-list
 Name              State    Autostart   Persistent
----------------------------------------------------
 crc               active   no          no
```

Reply:
You said 
> @cfergeau this check is only present if a user manually delete the ~./crc or ~/.crc/machine directory and don't want to use crc anymore so he can clean up those using cleanup command.

I was answering to that, removing `~/.crc` does not change anything with respect to libvirt `crc` network being there or not. I'm not asking for any change, but I don't understand why we need to check for `crc` network existence.